vagrant/windows/README.md

```diff
@@ -47,3 +47,14 @@ make build
 ```
 
 to build the binary.
+
+## Building Windows containers
+
+Use the [docker](docker/) machine, as it comes with Docker for Windows installed and configured. After building
+the application, you can build a Windows container the following way:
+
+```bash
+cp otelcolbuilder/cmd/otelcol-sumo.exe .
+make build-push-container-windows-dev PLATFORM=windows/amd64/ltsc2022
+docker run public.ecr.aws/sumologic/sumologic-otel-collector-dev:latest-windows-amd64-ltsc2022 --version
```

Comment:
```
+ docker build --file Dockerfile_windows --build-arg BUILD_TAG=latest --build-arg BASE_IMAGE_TAG=ltsc2022 --build-arg BUILDKIT_INLINE_CACHE=1 --platform=windows/amd64 --tag public.ecr.aws/sumologic/sumologic-otel-collector-dev:latest-windows-amd64-ltsc2022 .
unable to prepare context: unable to evaluate symlinks in Dockerfile path: CreateFile \\vboxsvr\sumologicDockerfile_windows: The network name cannot be found.
make: *** [Makefile:302: _build-container-multiplatform] Error 1
```

Reply:
Have you modified the Vagrantfile? That looks like a shared folder problem of some kind. Does it work if you copy the repo to the VM disk?

Reply:
It works when I change workdir to `\\vboxsrv\sumologic\`, and also works if using `bash`

Reply:
Allright, can we merge it as-is then, and work through this problem afterwards?

Reply:
Fine for me as I know the solution :)